src/ui/RadioButtonGroup.js

```diff
@@ -15,7 +15,9 @@ const RadioButtonGroup = ({
 }) => {
   return (
     <Stack className={className} as="div" spacing={3} {...getStackProps(props)}>
-      {groupLabel && <Label variant={LabelVariants.BOLD}>{groupLabel}</Label>}
+      {groupLabel ? (
+        <Label variant={LabelVariants.BOLD}>{groupLabel}</Label>
+      ) : null}
```

Comment:
```suggestion
      {groupLabel && <Label variant={LabelVariants.BOLD}>{groupLabel}</Label>}
```
Btw this code would also work, as it was. Conditions resulting in false are not rendered by React

Reply:
I tried that approach first but then I got this error
```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

Reply:
It's because it can return `false`, which a Stack/Inline can't handle